.couch_email_authrc-dist

```diff
@@ -0,0 +1,22 @@
+host = 127.0.0.1
+port = 3000
+couchDbBaseUrl = http://admin:secret@localhost:5984
+usersDb = _users
+
+; the time in seconds the authentication token
+; (which is part of the sign-in link) is valid
+tokenExpiryTime = 1
```

Comment:
Your description in the comment is great, but the value is still a bit too low ;)
Something like `3600` (i.e., one hour) would be reasonable, I guess.